minerals/startlegcord

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# Set FEATURES to what goes after --enable-features; for example, UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc will then get turned into --enable-features=UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc.
+FEATURES="UseSkiaRenderer"
+TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID:-app.legcord.Legcord}"
+declare -a FLAGS=(--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --disable-gpu-blocklist)
+
+if [[ $XDG_SESSION_TYPE == "wayland" ]]
```

Comment:
how does this behave when run on a wayland session?

Reply:
If Wayland socket is available, it applies neccesary flags for screenshare to work + flags to allow HW accel.